app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java

```diff
@@ -64,14 +65,35 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         AppUtil.setTaskDescription(activity, session.title, ContextCompat.getColor(activity, session.category.getVividColorResId()));
     }
 
+    // Receive all viewModel's properties changed.
+    private final Observable.OnPropertyChangedCallback onPropertyChanged = new Observable.OnPropertyChangedCallback() {
```

Comment:
Hmm, it looks DataBinding way 📝 
I guess EventBus seems better. To be honest, I want to use `setResult` and `onActivityResult`.
So how do you think this way?
https://github.com/konifar/droidkaigi2016/pull/385/files#diff-34844144bceb39cb50e2efdb17cabe71R85